### ISSUE TITLE
uncaught exception related to google chrome autoplay policy changes

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#4855]
### Observed behavior

We've started getting errors like:

> **NotAllowedError**
> `play()` failed because the user didn't interact with the document first.
> https://goo.gl/xX8pDD

### Expected behavior

* We should investigate the new policy changes and figure out what it means for our UX (cc @jtamiace @khangmach)
* We should ensure that there are no uncaught exceptions

### User-facing consequences

Video autoplay behaviors don't behave as designed

### Errors and logs

```
play() failed because the user didn't interact with the document first.
Uncaught (in promise) DOMException
```

### Steps to reproduce

This appears to happen when you navigate _directly_ to a video by pasting a link in, e.g. this one:

```
http://kolibribeta.learningequality.org/learn/#/topics/c/6189535e772c4a5abebe1329eb88d7ab
```

### Context

0.12.0 alpha 7

**Comment by rtibbles** [comment]
When I upgraded video-js I was seeing this a lot, but couldn't find a way to catch it, because it happens during the video js player initialization (and they seem to have designed it this way, and it doesn't seem to affect the running of video-js). Further, as soon as you do any DOM interaction, video-js then starts playing, as that is all is required to unblock Chrome's autoplay policy.

I think the current behaviour is the best we can do (I also experimented with using the video-js 'muted' flag for autoplay, but that seemed a more confusing user experience, with a less salient route to unblocking than pressing play).
